src/genesys/toploc/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .poly import ProofPoly, build_proofs, build_proofs_base64  # noqa: F401
```

Comment:
```suggestion
from genesys.toploc.poly import ProofPoly, build_proofs, build_proofs_base64  # noqa: F401
```

lets use absolute import ?

Reply:
Don't you think that it makes sense to create a separate package for toploc ? 

or maybe we incub it here for now

Reply:
Yea I think it makes sense to be a separate package. the validator uses it too